app/controllers/api/v1/doorkeeper/profiles_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::Doorkeeper::ProfilesController < Api::BaseController
+  before_action :doorkeeper_authorize!
+
+  def show
+    profile_info = JSON.parse(current_user.to_json).merge('api_key' => current_user.api_key)
```

Comment:
You can use `as_json` here to return a hash and then you won't need to parse JSON.